repo/packages/C/cassandra/0/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cassandra",
-  "version": "0.1.0-SNAPSHOT",
+  "version": "0.1.0-SNAPSHOT-396-rewrite-c0e080b4d4",
```

Comment:
Just FYI, it would be great to not have snapshot builds specified by packages for repeatability.  Is this tag ephemeral, or is the SNAPSHOT part of the version misleading.